static/icons/scalable/cartaviewer.svg

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
wonder if you should add `static/icons/** -linguist-generated` to the `.gitattributes` file, so that they don't show up as text diffs?

Reply:
I think we need to handle the SVG icons and the PNG icons differently. The SVGs should be marked as binary, so that they are excluded from text diffs. The PNGs are already detected as binary, but I suppose that we can mark them as generated, since they are programmatically generated from the full-colour SVG. I will add this.

Reply:
That did not do what I expected. :)

Reply:
Marking the SVGs as binary didn't seem to have any effect on the PR diff, but marking them as generated does work, so I guess we can go with that (and the SVGs themselves aren't hand-coded either, so I guess this makes sense).

Reply:
github seems to explicitly ignore the binary attribute in `gitattributes` and instead rely on `linguist` :thinking: